includes/class-wc-stripe-intent-controller.php

```diff
@@ -1029,7 +1035,7 @@ public function create_and_confirm_setup_intent_ajax() {
 
 			$setup_intent = $this->create_and_confirm_setup_intent( $payment_information );
 
-			if ( empty( $setup_intent->status ) || ! in_array( $setup_intent->status, [ 'succeeded', 'processing', 'requires_action' ], true ) ) {
+			if ( empty( $setup_intent->status ) || ! in_array( $setup_intent->status, [ 'succeeded', 'processing', 'requires_action', 'requires_confirmation' ], true ) ) {
```

Comment:
Because we're no longer confirming Cash App setup intents immediately we need to add `requires_confirmation` as a valid status we're expecting.